ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17.yaml

```diff
@@ -575,38 +623,38 @@ tests:
       BRANCH: "4.17"
       TEST_TYPE: jenkins-e2e-rhel-only
     workflow: openshift-e2e-libvirt-upi
-- as: ocp-e2e-serial-ovn-remote-libvirt-ppc64le
+- as: ocp-e2e-serial-ovn-remote-libvirt-multi-p-p
   cluster: build01
   cron: 0 18 * * 6
   steps:
     cluster_profile: libvirt-ppc64le
     dependencies:
-      OPENSHIFT_INSTALL_TARGET: release:ppc64le-latest
+      OPENSHIFT_INSTALL_TARGET: release:multi-latest
     env:
       ARCH: ppc64le
       BRANCH: "4.17"
       TEST_TYPE: conformance-serial
     workflow: openshift-e2e-libvirt-upi
-- as: ocp-fips-ovn-remote-libvirt-ppc64le
+- as: ocp-fips-ovn-remote-libvirt-multi-p-p
   cluster: build01
   cron: 0 19 * * 6
   steps:
     cluster_profile: libvirt-ppc64le
     dependencies:
-      OPENSHIFT_INSTALL_TARGET: release:ppc64le-latest
+      OPENSHIFT_INSTALL_TARGET: release:multi-latest
     env:
       ARCH: ppc64le
       BRANCH: "4.17"
       FIPS_ENABLED: "true"
       TEST_TYPE: conformance-parallel
     workflow: openshift-e2e-libvirt-upi-fips
-- as: ocp-e2e-compact-ovn-remote-libvirt-ppc64le
+- as: ocp-e2e-compact-ovn-remote-libvirt-multi-p-p
```

Comment:
This name is weird because compact might be referring to the test suite, but more likely it's supposed to be a "deployment type" like "sno"